qa/tasks/cephfs/test_cephfs_shell.py

```diff
@@ -137,13 +160,13 @@ def test_put_and_get_without_target_directory(self):
         Test that put fails without target path
         """
         # generate test data in a directory
-        self._cephfs_shell("!mkdir p1")
-        self._cephfs_shell('!dd if=/dev/urandom of=p1/dump1 bs=1M count=1')
-        self._cephfs_shell('!dd if=/dev/urandom of=p1/dump2 bs=2M count=1')
-        self._cephfs_shell('!dd if=/dev/urandom of=p1/dump3 bs=3M count=1')
+        self.run_cephfs_shell_cmd("!mkdir p1", self.mount_a)
+        self.run_cephfs_shell_cmd('!dd if=/dev/urandom of=p1/dump1 bs=1M count=1', self.mount_a)
+        self.run_cephfs_shell_cmd('!dd if=/dev/urandom of=p1/dump2 bs=2M count=1', self.mount_a)
+        self.run_cephfs_shell_cmd('!dd if=/dev/urandom of=p1/dump3 bs=3M count=1', self.mount_a)
```

Comment:
These can be collected into a single batch script now, or that should be possible. i.e.:

````
self.run_cephfs_shell_cmd("""
!mkdir ...
!dd ...
""", self.mount_a)
````

Reply:
In my experience from working with test_volume_client.py, I feel it's better to issue commands individually rather than in batches. Former, makes it easier to debug the tests.